Update boolean.md (#615)
Codecademy/docs: content/general/concepts/boolean/boolean.md

- `>=` and `<=`, for "greater than" and "less than", respectively.
- `==` and `is` or `!=` and `not` to test for equality and inequality, respectively.

> Other languages may use "falsy" values like `""`, `null`, or `0`. And "truthy" values such as `1`. Some even use a combination of both approaches.

Reviewer: ```suggestion
Other languages may use a combination of "falsy" values (e.g. `""`, `null`, or `0`) and "truthy" values such as `1`.
```

Actually what do we think of this suggestion?

Author: @Dusch4593 I like it. It takes away unnecessary words and makes it simpler.
